ADPhantomApp/src/ADPhantom.cpp

```diff
@@ -2227,6 +2237,37 @@ asynStatus ADPhantom::writeInt32(asynUser *pasynUser, epicsInt32 value)
       status |= setCameraParameter("auto.bref", value);
     } 
   }
+  else if (function == PHANTOM_DataFormat_){
+    //Update the selected token and associated bit depth
+      if (value==0){
+        bitDepth_ = 8;
+        phantomToken_ = "8";
+      }
+      else if (value==1){
+        bitDepth_ = 8;
+        phantomToken_ = "8R";
+      }
+      else if (value==2){
+        bitDepth_ = 16;
+        phantomToken_ = "P16";
+      }
+      else if (value==3){
+        bitDepth_ = 16;
+        phantomToken_ = "P16R";
+      }
+      else if (value==4){
+        bitDepth_ = 10;
+        phantomToken_ = "P10";
+      }
+      else if (value==5){
+        bitDepth_ = 12;
+        phantomToken_ = "P12L";
+        printf("P12L selected!");
+      }
```

Comment:
Due to the mis-match with the mbbo this won't actually ever be selected (also likely want to remove the P12L selected print)